ubirch/ubirch_protocol.go

```diff
@@ -325,3 +326,27 @@ func (p *Protocol) Verify(name string, upp []byte) (bool, error) {
 		return false, fmt.Errorf("invalid protocol version: 0x%02x", upp[1])
 	}
 }
+
+// CheckChain compares the signature bytes of a previous ubirch-protocol message with the previous signature bytes of
+// a second ubirch-protocol message and returns true if they match. Returns an error if one of the UPPs is invalid.
+func CheckChain(prev []byte, chained []byte) (bool, error) {
```

Comment:
the names `prev` and `chained` are not very descriptive. This can lead to misunderstanding. Maybe `prevUpp` and `currentUpp` or `recentUpp`, something like that?

Reply:
how does `previousUPP` and `consecutiveUPP` sound? I am just thinking that you can also verify the chain between two older UPPs with this method.